### PR TITLE
Update `host` and `host_section` docs

### PR DESCRIPTION
**Pull request by geoffreymm**

### CODE REVIEW

**Comment by geoffreymm** [comment]
[Update drops documentation on easol.dev](https://trello.com/c/glhZWKqz/53-update-drops-documentation-on-easoldev)

File: docs/reference/objects/product/host.md [hunk 1]
```diff
@@ -4,25 +4,29 @@ title: Host
 parent: Product
 ---
 
-When using a `host` object you have access to the following attributes
-Can be accessed on a product like below
+When using a `host` object you have access to the following attributes.<br>
+Can be accessed on a product like below:
```
Comment (ianmooney) [inline docs/reference/objects/product/host.md:8]: Maybe remove the `<br>Can be accessed on a product like below:` part, as there's no example given.

File: docs/reference/objects/product/host.md [hunk 2]
```diff
@@ -4,25 +4,29 @@ title: Host
...
 
 # host.body
 
 Returns the body of the host, this is a rich text field and so will return html as in the example above.
```
Comment (ianmooney) [inline docs/reference/objects/product/host.md:12]: Maybe remove `as in the example above.`, as there is no example.

File: docs/reference/objects/product/host_section.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Host Section
+parent: Product
+---
+
+When using a `host_section` object you have access to the following attributes.<br>
+Can be accessed on a product like below
```
Comment (ianmooney) [inline docs/reference/objects/product/host_section.md:8]: Same here, perhaps remove `Can be accessed on a product like below`.

File: docs/reference/objects/product/host_section.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+
+# host_section.overview
+
+Returns the overview for the host section.
+
+
+# host_section.elements
```
Comment (ianmooney) [inline docs/reference/objects/product/host_section.md:16]: Can we swap these attributes so they are alphabetical?

**Review by ianmooney — COMMENTED** [review]
Nice one - just left a few small comments.

**Comment by geoffreymm** [comment]
All great shouts @ianmooney thanks. Should be good to go.

**Comment by geoffreymm** [comment]
[Update drops documentation on easol.dev](https://trello.com/c/glhZWKqz/53-update-drops-documentation-on-easoldev)

**Review by ianmooney — APPROVED** [review]
Nice 👍

**Pull request merged**